SPECS/cloud-init/mariner-22.1.patch

```diff
@@ -103,17 +105,32 @@ diff -Naur a/cloudinit/distros/mariner.py b/cloudinit/distros/mariner.py
 +
 +LOG = logging.getLogger(__name__)
 +
++NETWORK_FILE_HEADER = """\
++# This file is generated from information provided by the datasource. Changes
++# to it will not persist across an instance reboot. To disable cloud-init's
++# network configuration capabilities, write a file
++# /etc/cloud/cloud.cfg.d/99-disable-network-config.cfg with the following:
++# network: {config: disabled}
++"""
 +
 +class Distro(distros.Distro):
 +    systemd_hostname_conf_fn = '/etc/hostname'
 +    network_conf_dir = '/etc/systemd/network/'
 +    systemd_locale_conf_fn = '/etc/locale.conf'
 +    resolve_conf_fn = '/etc/systemd/resolved.conf'
 +
++    network_conf_fn = {
++        "netplan": "/etc/netplan/50-cloud-init.yaml"
```

Comment:
Should this file be available in any of our packages? I see it in neither 1.0 nor 2.0, so I'm guessing it might be created manually/during image build but I wanted to double-check we've migrated everything we need from 1.0.

Reply:
This filename is a hint to cloud-init for where it should place the config that it passes to netplan. It's dynamically created at runtime. Thanks for double-checking!